src/openms/source/FORMAT/MzTab.cpp

```diff
@@ -814,9 +825,15 @@ namespace OpenMS
     const vector<PeptideIdentification>& pep_ids = f.getPeptideIdentifications();
     if (pep_ids.empty())
     {
+      // still add empty opt_ columns before returning
+      addMetaInfoToOptionalColumns(peptide_identifications_user_value_keys, row.opt_, String("global"), MetaInfoInterface());
+      addMetaInfoToOptionalColumns(peptide_hit_user_value_keys, row.opt_, String("global"), MetaInfoInterface());      
```

Comment:
```suggestion
      addMetaInfoToOptionalColumns(peptide_identifications_user_value_keys, row.opt_, "global", MetaInfoInterface());
      addMetaInfoToOptionalColumns(peptide_hit_user_value_keys, row.opt_, "global", MetaInfoInterface());      
```

Reply:
the compiler will take care of the conversion.
Makes it a bit more readable, and prevents double conversion when changing the function's signature (to use std::stringview, for example).